src/components/MyProfile.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import '../profile.css';
+
+const MyProfile = () => {
+  const userReservedRockets = useSelector((
+    state,
+  ) => state.rockets.filter((rocket) => rocket.reserved));
+
+  return (
+    <div className="profile-container">
+      <h2>My Profile</h2>
+      <div className="profile-content">
+        <div className="reserved-rockets">
+          <table>
+            <thead>
+              <tr>
+                <th>Rocket Title</th>
+              </tr>
+            </thead>
+            <tbody>
+              {userReservedRockets.map((rocket) => (
+                <tr key={rocket.id}>
+                  <td>{rocket.title}</td>
+                </tr>
+              ))}
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+  );
```

Comment:
Please consider replacing the 'My Profile' label with 'My Rockets' and implementing a border around the rocket names. Additionally, when no rockets are reserved, it would be beneficial to display 'No rockets reserved yet!' as an alternative to 'Rocket Title.' Below is an illustrative code snippet:

```js
import React from 'react';
import { useSelector } from 'react-redux';
import '../profile.css';

const MyProfile = () => {
  const userReservedRockets = useSelector((state) =>
    state.rockets.filter((rocket) => rocket.reserved)
  );

  return (
    <div className="profile-container">
      <h2>My Rockets</h2>
      <div className="profile-content">
        <div className="reserved-rockets">
          {userReservedRockets.length > 0 ? (
            <table>
              <thead>
                <tr>
                  <th>{' '}</th>
                </tr>
              </thead>
              <tbody>
                {userReservedRockets.map((rocket) => (
                  <tr key={rocket.id}>
                    <td style={{ border: '1px solid #ccc' }}>{rocket.title}</td>
                  </tr>
                ))}
              </tbody>
            </table>
          ) : (
            <p>No rockets reserved yet!</p>
          )}
        </div>
      </div>
    </div>
  );
};

export default MyProfile;

```